.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "yaml-test-suite/testdata/data-2022-01-17"]
+	path = yaml-test-suite/testdata/data-2022-01-17
```

Comment:
What is the reason behind usage of a path about 2022-01-17 ?

I mean if there is a need to state it's old, using 2022 would be enough, no?

To me here it looks odd.

Or maybe there is a need for another folder, now or later that will contain a new test suite.

But for now, I wouldn't use a date in the path.

Reply:
Naming is a pain 😅

Let's see if someone among the reviewers has an idea